docs/autos.md

```diff
@@ -0,0 +1,119 @@
+2. *Left side of the two-ball tarmac.* Give robot in position one a clear path to our field ball. Bump the opposition's field ball toward our hangar.
+3. **Right side of the one-ball tarmac, our preferred starting position for a zero-ball auto.** Give robot in position four a clear path to our hangar ball. Bump the opposition's field ball toward our hangar, taking care not to bump it into our field ball or toward our terminal ball.
+4. *Left side of the one-ball tarmac.* Give robot in position three a clear path to our hangar ball. Pin the opposition's hangar ball against the wall.
+
+# One-Ball Auto
+
+In one-ball auto we shoot the ball we start with, but we do not pick up any additional balls. The most likely reason we would run a one-ball auto is that our intake is broken.
+
+- *Shoot and move.* Everything is the same as zero-ball auto, except that we shoot our ball before taxiing out of the tarmac.
+- *Shoot and stay.* We shoot our ball and stay in place. The most likely reason to run "One-Ball Auto Shoot and Stay" is that our drive train is broken.
+
+# Two-Ball Auto
+
+In two-ball auto we shoot the ball we start with, move to pick up a ball, and shoot that ball. The most likely reason we would run a two-ball auto is that an alliance partner has a better three-ball or four-ball auto than ours. There are five objectives for two-ball auto:
+
+1. Score four points by shooting a ball into the high goal.
+2. Score two points by moving our robot completely out of the tarmac.
+3. Pick up a ball.
+4. Shoot that ball into the high goal.
+5. Stay out of the way of our alliance partners so they can score more points.
+
+# Three-Ball Auto
+
+In three-ball auto, we have six objectives:
+
+1. Score four points by shooting a ball into the high goal.
+2. Score two points by moving our robot completely out of the tarmac.
+3. Pick up a ball.
+4. Shoot that ball into the high goal.
+5. Pick up another ball.
+6. Shoot that ball into the high goal.
+
+# Four-Ball Auto
+
+In four-ball auto, we start in position one (right side of two-ball tarmac) and have eight objectives:
+
+1. Score four points by shooting a ball into the high goal.
+2. Score two points by moving our robot completely out of the tarmac.
+3. Pick up our wall ball.
+4. Shoot that ball into the high goal.
+5. Pick up our field ball.
+6. Shoot that ball into the high goal.
+7. Pick up our terminal ball.
+8. Shoot that ball into the high goal.
+
+# Five-Ball Auto
+
+In five-ball auto, we have to count on our human player to score or add abilities to our robot.
+
+To score the ball the human holds during auto:
+
+1. Human player could throw ball directly into hub.
+2. Human player could roll ball into robot. This requires a ramp intake on the robot.
+3. Human player could roll ball onto playing field. This requires robot to either position itself so human player can aim for ground-ball intake, or robot to have vision, ball detection, and "move to intake moving ball" capabilities.
```

Comment:
Human players can roll the ball under the human player station to avoid bounding. This is what Citrus Circuits did in their 5 ball auto. (Yes, they have already created one)